ldap-api/src/main/java/ca/bc/gov/hlth/ldapapi/service/LdapService.java

```diff
@@ -27,6 +28,8 @@
     private final String LDAP_CONST_ACCOUNT_LOCKED_ATTRIBUTE = "acctlockedflag";
     private final String LDAP_ATTR_ACCT_LOCKED_BY = "acctlockedby";
     private final String LDAP_ATTR_ACCT_LOCKED_REASON = "acctlockedreason";
+    public  final String LDAP_ATTR_PASSWORD_CHANGE_DATE = "passwordchangedate";
+    public static final String LDAP_ATTR_PASSLIFESPAN = "passlifespan";
     private final String LDAP_SEARCH_BASE = "o=hnet,st=bc,c=ca";
```

Comment:
Just a little style nitpick, but it looks like these can all be `private static final`?